README.md

```diff
@@ -67,6 +67,17 @@ of NIST-standardized elliptic curves and symmetric cryptosystems.
 [Stanford Pairing-Based Crypto (PBC) library](http://crypto.stanford.edu/pbc/),
 which will of course only work if the PBC library is installed.
 
+Issues
+------
+
+- Traditionally, ECDH (Elliptic curve Diffie-Hellman) derives the shared secret
+from the x point only. In this framework, you can either manually retrieve
+the value or take the combined (x, y) value. The latter is the recommended
+process as it can be done without loss of security and done more cleanly,
```

Comment:
Avoid passive voice in technical writing!  (ALWAYS!!! :) )  "is the recommended process" - recommended by whom, and in what context?  And why can the latter be done more cleanly?  I would say:

```...or use the MarshalBinary method to take the combined (x, y) value as the shared secret.  We recommend the latter process for new software/protocols using this framework as it is cleaner and generalizes across different types of groups (e.g., both integer and elliptic-curve groups), ...```